docs/source/docker/advanced/datasource_predictions.rst

```diff
@@ -13,6 +13,9 @@ add their predictions to the datasource.
 If you already have labels instead of predictions, you can treat them
 just as predictions and upload them the same way.
 
+.. warning:: Note that using predictions requires a minimum 
+    docker of version 2.2. You can check your docker version by running the 
```

Comment:
its not called "docker" :)
Note that using predictions requires the worker docker image of version 2.2.0 or higher